ai-eval-service/main.py

```
@@ -87,14 +98,13 @@ def run_worker(payload: dict, timeout: int = 300):
                     f"Worker stdout (first 500 chars): {result.stdout[:500]}. "
                     f"Worker stderr (first 500 chars): {result.stderr[:500]}"
                 )
-                # Re-raise with context to ensure the parsing error isn't silently ignored
                 raise RuntimeError(
                     f"Worker process failed and error response could not be parsed as JSON. "
                     f"Worker stderr: {result.stderr[:500]}"
                 ) from e
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)